exasol_script_languages_developer_sandbox/lib/aws_access/aws_access.py

```diff
@@ -200,6 +200,19 @@ def export_ami_image_to_vm(self, image_id: str, tag_value: str,
         if status != "completed":
             raise RuntimeError(f"Export of VM failed: status message was {status_message}")
 
+    def get_ami(self, image_id: str) -> Any:
```

Comment:
I thought we get Dict[str,Any]

Reply:
this was changed in a later commit. Currently it returns a fascade object